File: week6/sentimental-mario-less/mario.py


def main():
    height = get_height()
    # printng rows
    for i in range(height):
        # printing inner loop with .
        x = int(height - i - 1)
        for j in range(x):
            print(" ", end="")
        # inner loops with #
        y = int(height - x)
        for k in range(y):
            print("#", end="")
        print()

# get size of grid


def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                return n
        except ValueError:
            print("Error")
            

main()

File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get size of grid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Print # and . and " "
    //Loop for rows
    for (int i = 0; i < n; i++)
    {
        //Inner loop for . and " "
        for (int x = 0; x < n - i - 1; x++)
        {
            printf(" ");
        }
        //Inner Loop for #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}


